src/datepicker/constants.js

```diff
@@ -27,3 +27,8 @@ export const STATE_CHANGE_TYPE = Object.freeze({
 export const WEEKDAYS = [0, 1, 2, 3, 4, 5, 6];
 
 export const DEFAULT_MONTHS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+
```

Comment:
Please suggest alternatives to the variable names I've used. In Figma the term "Medium" corresponds with what I've called "default" here. According to Josh there may be other levels of density for the calendar, so I chose not to use a boolean for this prop.